tests/check_merge_conflict_test.py

```diff
@@ -5,11 +5,11 @@
 import shutil
 
 import pytest
+from py.path import local
```

Comment:
me too, also I don't think this is shippable because it breaks windows.  Stupid default newline handling. marley lod py.path